test/e2e/performanceprofile/functests/4_latency/latency.go

```diff
@@ -463,17 +450,26 @@ func createLatencyTestPod(testPod *corev1.Pod) {
 	Expect(err).ToNot(HaveOccurred(), "pod %q did not reach %q phase; error: %v", podKey, corev1.PodSucceeded, err)
 }
 
-func extractLatencyValues(exp string, pod *corev1.Pod) string {
+func extractLatencyValues(exp string, pod *corev1.Pod) []int {
 	out, err := pods.GetLogs(testclient.K8sClient, pod)
 	Expect(err).ToNot(HaveOccurred())
 
 	maximumRegex, err := regexp.Compile(exp)
 	Expect(err).ToNot(HaveOccurred())
 
-	latencies := maximumRegex.FindStringSubmatch(out)
-	Expect(len(latencies)).To(Equal(2))
+	output := maximumRegex.FindStringSubmatch(out)
+	Expect(len(output)).To(Equal(2))
 
-	return latencies[1]
+	var latencies []int
+	for _, lat := range strings.Split(output[1], " ") {
+		if lat == "" {
+			continue
+		}
+		curr, err := strconv.Atoi(lat)
+		Expect(err).ToNot(HaveOccurred())
```

Comment:
In case of asserting on helper function, please add `GinkgoHelper()` at the beginning of the function